packages/camltc/camltc.0.9.8/files/osx.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/dune b/src/dune
+index be85e03..05d61a4 100644
+--- a/src/dune
++++ b/src/dune
+@@ -9,6 +9,7 @@
+  (flags -w -27)
+  (c_names otc_wrapper)
+  (c_flags -O2 -I 3rd-party/tokyocabinet)
++ (modes native)
```

Comment:
I am surprised that this is needed, I thought `native` was the default value

Reply:
I think native is the default indeed, but as I dont know how to build a shared lib under osx (I dont have access to mac), I believe this will disable it. Anyways it will not compile without it.

Reply:
just to elaborate a bit: Under linux, If I remove the rule to generate the 'dlltc_stubs.so', the build will fail. Adding `(modes native)` to the library camltc.tc make is complete, and I assume the same goes for the osx build. 
I would agree that the `(modes native)` is slightly redundant for the camltc library, as I assume its inferred from camltc.tc lib, but is causes no harm.